GeneralStateTests/stCallCodes/EOF1_Execution.json

```diff
@@ -0,0 +1,1048 @@
+                },
+                {
+                    "hash" : "0xe4793415be5f6093d3b397ac5fc98b07df48418fedbe2071926c8bcab9c2bee3",
+                    "indexes" : {
+                        "data" : 28,
+                        "gas" : 0,
+                        "value" : 0
+                    },
+                    "logs" : "0x1dcc4de8dec75d7aab85b567b6ccd41ad312451b948a7413f0a142fd40d49347",
+                    "txbytes" : "0xf874800a830aae6094b94f5374fce5edbc8e2a8697c15331677e6ebf0b8094624f5374fce5edbc8e2a8697c15331677e6ebf0b1ca05d3d57e2e62661735a1b51901f4ee7e6af4cb4a5b4f307fe7dae5b1082935c40a04e89f86c38132944ad4d92a4d7b383e976abd7c676a5981e0f13a6cbc7fdf07d"
+                },
+                {
+                    "hash" : "0x1a3f19624157bdbc2fb8dc2dd261d048a02ce88292dbf03627aa4ec4aa813422",
+                    "indexes" : {
+                        "data" : 29,
+                        "gas" : 0,
+                        "value" : 0
+                    },
+                    "logs" : "0x1dcc4de8dec75d7aab85b567b6ccd41ad312451b948a7413f0a142fd40d49347",
+                    "txbytes" : "0xf874800a830aae6094b94f5374fce5edbc8e2a8697c15331677e6ebf0b8094634f5374fce5edbc8e2a8697c15331677e6ebf0b1ba06c182d4bd2987caa55a7e1c4c8652d15e125d0fc8bb28ed2c57cb8e500655462a03ec200e4f10ab33e4d3e414a88883d47faaa1eabbf58c26c01a90fd8444f5c36"
+                },
+                {
+                    "hash" : "0x4ba63c7dc8368bff0bb515455b116bb9b4d236cf8fb8aa0214879aff4cf65d45",
+                    "indexes" : {
+                        "data" : 30,
+                        "gas" : 0,
+                        "value" : 0
+                    },
+                    "logs" : "0x1dcc4de8dec75d7aab85b567b6ccd41ad312451b948a7413f0a142fd40d49347",
+                    "txbytes" : "0xf874800a830aae6094b94f5374fce5edbc8e2a8697c15331677e6ebf0b8094644f5374fce5edbc8e2a8697c15331677e6ebf0b1ca056dd2833f1a7aac465f96308f049f18de8a0df6e18f9864664f41e5b553fc7c6a0455e0d8361e7839de2f16e4bf0ca9b196b75f32d9994524f67b4cc40a331c4de"
+                },
+                {
+                    "hash" : "0xef57395b4856eac81327501e8f4b41d1dde23dc1410a47b6b3a750dc7defa675",
+                    "indexes" : {
+                        "data" : 31,
+                        "gas" : 0,
+                        "value" : 0
+                    },
+                    "logs" : "0x1dcc4de8dec75d7aab85b567b6ccd41ad312451b948a7413f0a142fd40d49347",
+                    "txbytes" : "0xf874800a830aae6094b94f5374fce5edbc8e2a8697c15331677e6ebf0b8094654f5374fce5edbc8e2a8697c15331677e6ebf0b1ba08b6504c574b41e0d661a61accc0f42ea8c5672fa6911bb76c5f4e42eecca553aa049a5c5fc75958c321ffec2733712582156027b649c3d3aeee5f6becdfa79eeca"
```

Comment:
As above, this test is failing in the EthereumJS implememtation due to the code stored in 0x654f5374fce5edbc8e2a8697c15331677e6ebf0b having a computed container size of 43 bytes while the actual bytecode length is 42.

Reply:
In this case found the problem, you're right, code is invalid, will regenerate the test.

(We should work on making geth test generation tool reject invalid EOF in pre-state, then I'll catch all such problems)

Reply:
This case should be fixed now.